olive/passes/olive_pass.py

```diff
@@ -48,6 +50,8 @@ class Pass(ABC):
     _requires_user_script: bool = False
     # True if pass configuration requires data configuration which will leverage data container for pass execution
     _requires_data_config: bool = False
+    # True if pass configuration requires evaluation configuration
```

Comment:
better not use a class level field for eval customization since only inc_quantization only use it from the time being. 

The other two _requires_xxx is user by all passes.